functions/utils/database.js

```diff
@@ -8,8 +8,8 @@ const queryDocument = (collection, id) => {
   return db.collection(collection).doc(id).get();
 };
 
-const setDocument = (collection, id, document) => {
-  return db.collection(collection).doc(id).set(document);
+const setDocument = (collection, id, fields) => {
+  return db.collection(collection).doc(id).set(fields);
 };
```

Comment:
Any reason for this renaming?

Reply:
Yeah, I wrote that utility function a month or two ago and the naming annoyed me when I saw it again